source/braille.py

```diff
@@ -1394,6 +1394,40 @@ def _addFieldText(self, text, contentPos, separate=True):
 		self.rawTextTypeforms.extend((louis.plain_text,) * textLen)
 		self._rawToContentPos.extend((contentPos,) * textLen)
 
+	def _getBrailleFormatConfig(self, formatConfig):
+		"""
+		Filter format configuration to only include attributes needed for braille.
+		This significantly improves performance by reducing unnecessary formatting queries,
+		especially in Microsoft Word.
+
+		Braille displays don't need visual formatting like fonts, colors, etc.
+		Only structural information (headings, lists, links) is relevant.
+		"""
+		# Create a shallow copy to avoid modifying the original
+		brailleFormatConfig = dict(formatConfig)
+
+		# Disable visual formatting attributes that braille doesn't use
+		unnecessaryAttrs = [
```

Comment:
Have you verified that these attributes will never show in Braille?

CC @LeonarddeR, @nvdaes.

Reply:
I'm pretty sure we support reporting many of these. I also find it hard to believe that these effect latency